### DISCUSSION TITLE
Support for setting tile size programmatically?

### DISCUSSION THREAD

**Discussion by eladrkatz** [DevbookHQ/splitter#6] [Ideas]
Settings size or toggling open/closed? 
How would I approach adding that?

Thanks!

**Comment by mlejva** [comment]
Hello @eladrkatz, you can set the **initial** size of Splitter using the `initialSizes` prop. An example is shown [here](https://github.com/DevbookHQ/splitter/blob/master/example/src/InitialSizes/index.tsx).

Currently, there isn't a way to set the sizes of Splitter's child views programmatically.
